webtask-templates/visualstudio.json

```diff
@@ -1,7 +1,7 @@
 {
   "title": "Visual Studio Team Services Deployments",
   "name": "auth0-visualstudio-deploy",
-  "version": "2.6.2",
+  "version": "2.6.3",
   "preVersion": "2.5.3",
   "author": "auth0",
   "description": "This extension gives Auth0 customers the possibility to deploy Hosted Pages, Rules and Custom Database Connections from Visual Studio Team Services.",
```

Comment:
Thanks for open sourcing some of your code. Would it be possible to start writing more unit tests? I've noticed a few PRs have gone by now without them.

Reply:
Yep. Added that to my backlog.